tests/test_embeddings.py

```diff
@@ -32,6 +32,32 @@ def test_embeddings(embeddings_fifu, embeddings_text, embeddings_text_dims):
             embedding.embedding, storage_row), "FiFu and storage row  mismatch"
 
 
+def test_unknown_embeddings(embeddings_fifu):
+    assert embeddings_fifu.embedding("OOV") is None, "No for default failed"
```

Comment:
No -> None?

Or otherwise: Unknown lookup without default failed

Reply:
But there's the `None` default ;)

Reply:
No, there's no default, not enough coffee yet...